source/jormungandr/tests/proxy_realtime_tests.py

```diff
@@ -148,3 +150,50 @@ def test_departures(self):
             ("L", "20160102T101100"),
             ]
         eq_(departures, expected_departures)
+
+    def test_departures_realtime_informations(self):
+        query = 'stop_areas/S42/departures?from_datetime=20160102T1000&show_codes=true&count=7'
+        response = self.query_region(query)
+
+        assert "departures" in response
+        assert len(response["departures"]) == 7
+
+        dep_data_freshness = [(d['route']['name'], d['stop_date_time']['data_freshness'])
+                      for d in response['departures']]
+        expected_data_freshness = [
+            ("J", "realtime"),
```

Comment:
I think we can mutualize some code with my PR https://github.com/CanalTP/navitia/pull/1471